src/NpmBridge.php

```diff
@@ -97,6 +97,14 @@ public function isDependantPackage(
             }
         }
 
+        // Check custom TAO extension config and if it has NPM Bridge configuration, run assets bulding
+        if (
+            $package->getType() === 'tao-extension'
+            && array_key_exists('npm-bridge', $package->getExtra())
+        ) {
+            return true;
+        }
+
         return false;
```

Comment:
Could have been simplified into one return statement.

Reply:
In the current implementation, I would prefer to leave as is, because it has 3 clear conditions. If simplify, the whole method should be refactored.

Reply:
I meant only line 100-108:

```suggestion
        return $package->getType() === 'tao-extension'
            && array_key_exists('npm-bridge', $package->getExtra())
        }
```

It's no big deal though, for me this is easier to oversee :)